### PR TITLE
Adding videos

### PR DESCRIPTION
**Pull request by jthake-msft**

### CODE REVIEW

File: concepts/office365-groups-concept-overview.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ ms.prod: "reports"
 
 Office 365 groups provide the foundational membership service for users to share conversations, files, notes, calendars, plans, and many other assets. 
 
+<iframe src="https://www.youtube-nocookie.com/embed/WB9w6QM9xIU" width="640" height="320" allowFullScreen="true" frameBorder="0"></iframe>
```
Comment (Lauragra) [inline concepts/office365-groups-concept-overview.md]: I think you need to use the Markdown syntax, not the HTML: > [!VIDEO https://www.youtube.com/embed/iAtwVM-Z7rY]

Comment (jthake-msft) [reply concepts/office365-groups-concept-overview.md]: this is working fine https://review.docs.microsoft.com/en-us/graph/office365-groups-concept-overview?branch=jthake-videos as it gives me more control

Comment (jthake-msft) [reply concepts/office365-groups-concept-overview.md]: is there any way to do another line break before the video @Lauragra ?

Comment (Lauragra) [reply concepts/office365-groups-concept-overview.md]: You can use <br/> tags for line breaks. The video doesn't look like the example in the help topic. It's kind of stretched out and there's a black border on the sides. In mobile view, the black borders move to the top and bottom.

Comment (Lauragra) [reply concepts/office365-groups-concept-overview.md]: br tags

Comment (jthake-msft) [reply concepts/office365-groups-concept-overview.md]: the video is a set ratio dimension. the black borders will be there as the embed fills the page width and doesn't skew the video dimensions...

Comment (Lauragra) [reply concepts/office365-groups-concept-overview.md]: I did a test with the video syntax as prescribed in the help topic, and it doesn't render with the black borders: https://review.docs.microsoft.com/en-us/graph/office365-groups-concept-overview?branch=lauragra-test

Comment (jthake-msft) [reply concepts/office365-groups-concept-overview.md]: yes but then it doesn't sit center in the page. I personally prefer the video centered, than to the left...

Comment (Lauragra) [reply concepts/office365-groups-concept-overview.md]: I think it's better to have it be left-justified and not have the black bars when the video is played. Also, we should probably adhere to the prescribed syntax.

File: concepts/outlook-mail-concept-overview.md [hunk 2]
```diff
@@ -11,6 +11,10 @@ ms.prod: "outlook"
 Outlook is a messaging communication hub in Office 365. It also lets you manage contacts, schedule meetings, find information about users in an organization,
 initiate online conversations, share files, and collaborate in groups.
 
+
+> [!VIDEO https://www.youtube-nocookie.com/embed/gm25wusIQ]
```
Comment (Lauragra) [inline concepts/outlook-mail-concept-overview.md]: We need to fix this video link before we can merge. Everything else looks good on staging.

Comment (jthake-msft) [reply concepts/outlook-mail-concept-overview.md]: https://www.youtube-nocookie.com/embed/L-gm25wusIQ

**Pull request merged**